cmd/start.go

```diff
@@ -42,16 +42,13 @@ const (
 const startDescription = `Starts the Keep client in the foreground. Currently this only consists of the
    threshold relay client for the Keep random beacon.`
 
-// Constants related with balance monitoring.
-const (
-	// defaultBalanceAlertThreshold determines the alert threshold below which
-	// the alert should be triggered.
-	defaultBalanceAlertThreshold = 500000000000000000 // 0.5 ETH
-
-	// defaultBalanceMonitoringTick determines how often the monitoring
-	// check should be triggered.
-	defaultBalanceMonitoringTick = 10 * time.Minute
-)
+// Values related with balance monitoring.
+// defaultBalanceAlertThreshold determines the alert threshold below which
+// the alert should be triggered.
+var defaultBalanceAlertThreshold = big.NewInt(500000000000000000) // 0.5 ether
```

Comment:
We could have a `WeiFromString` function declared in `github.com/keep-network/keep-common/pkg/chain/ethereum` that would let us handle this gracefully as: `ethereum.WeiFromString("0.5 ether")`

Reply:
This would be nice, but I'm not sure if the usage could be that elegant as the function would also have return an error returned by `UnmarshalText`. Maybe we could investigate that in a follow-up task as this requires changes to the keep-common repository.

What's more, we could update functions signatures (also in contracts code generators) to expect `ethereum.Wei` type instead of `big.Int`.

Reply:
Yeah, agreed. It's a broader change. Right now, `ethereum.Wei` docs are clear about its usage:
> Wei is a custom type to handle Ether value parsing in configuration files

And we should expect nothing more. `var defaultBalanceAlertThreshold` is not a value from a configuration file and the way the conversion is handled later in this file is very graceful IMO.